test/integration/src/client-helper.js

```diff
@@ -1,14 +1,20 @@
 const configHelper = require('./config-helper');
-const revai = require('../../../dist/src/api-client');
+const revaiAsync = require('../../../dist/src/api-client');
+const revaiCustomVocabularies = require('../../../dist/src/custom-vocabularies-client');
 const JobStatus = require('../../../dist/src/models/JobStatus').JobStatus;
 const JobType = require('../../../dist/src/models/JobType').JobType;
 
 module.exports = {
-    getClient: (apiKey) => {
-        const client = new revai.RevAiApiClient(apiKey);
+    getAsyncClient: (apiKey) => {
+        const client = new revaiAsync.RevAiApiClient(apiKey);
         client.apiHandler.instance.defaults.baseURL = `https://${configHelper.getBaseUrl()}/revspeech/v1/`;
```

Comment:
While you're making these changes we might as well update this to be the newer url
```suggestion
        client.apiHandler.instance.defaults.baseURL = `https://${configHelper.getBaseUrl()}/speechtotext/v1/`;
```